io.openems.edge.io.shelly/src/io/openems/edge/io/shelly/shelly25/IoShelly25Impl.java

```diff
@@ -137,14 +137,19 @@ private void processHttpResult(HttpResponse<JsonElement> result, HttpError error
 		var relay1State = new RelayState(null, null, null);
 		var relay2State = new RelayState(null, null, null);
 
-		try {
```

Comment:
I had a NullPointerException whilst writing the Test - therefore we need to wrap this in a try catch.